pom.xml

```diff
@@ -267,6 +267,7 @@
 		<maven.compiler.source>${java.version}</maven.compiler.source>
 		<maven.compiler.target>${java.version}</maven.compiler.target>
 		<kotlin.compiler.jvmTarget>${java.version}</kotlin.compiler.jvmTarget>
+		<kotlin.compiler.version>2.2.21</kotlin.compiler.version>
```

Comment:
I am not a Kotlin expert, but what does it mean if you use Kotlin compiler 2.2.21? When I tried to compile the project using Kotlin version 1.9.25 it was complaining because Spring Framework 7 had classes that were compiled with 2.2. It could mean that users would have the same problem if they want to use Spring AI 1.x with Spring 6.